WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -1933,6 +1943,45 @@ extension AztecPostViewController {
         static let acceptTitle  = NSLocalizedString("Yes", comment: "Accept Action")
         static let cancelTitle  = NSLocalizedString("Not Now", comment: "Nicer dialog answer for \"No\".")
     }
+
+    struct ToolBarItemsAccessibility {
+        static let mediaLabel = NSLocalizedString("Insert media", comment: "Accessibility label for insert media button on formatting toolbar.")
+        static let mediaIdentifier = "format_toolbar_insert_media"
```

Comment:
@jleandroperez Maybe this will work better if I had an extension to the original FormattingIdentifier enum?